src/state/search/actions.js

```diff
@@ -30,9 +70,12 @@ export function fetchResults(searchTerm = '', pageNum = 1) {
 
 export function fetchResultsSucceeded(
   results,
+  filters,
   totalResults,
   currentPage,
-  searchTerm
+  searchTerm,
+  filterString,
+  appliedFilters
```

Comment:
hahahaha I asked you about this in the PR I just reviewed for you

Reply:
I really like the object approach actually. Since I'm going to rip out a lot of this when I take out the `appliedFilters` out of the reducer, I might convert it there then.